.eslintrc.json

```diff
@@ -14,6 +14,7 @@
     "prettier/prettier": [
       "error",
       {
+        "devDependencies": false,
```

Comment:
Don't disable es-lint errors without discussing it with the team. Now you disabled it project wide (e.g. nobody is getting errors about devDependencies now)

Reply:
Agreed